hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/models/UpdateTrackingAccount.java

```diff
@@ -38,28 +38,29 @@
 import org.hyperledger.besu.evm.account.MutableAccount;
 
 /**
- * A mutable and updatable implementation of the {@link MutableAccount} interface, that tracks account updates
- * since the creation of the updated it's linked to.
- *
- * Contains {@code updateAccountTracker} for immediate set of balance in the world state.
- * Note that in practice this only track the modified account values, but doesn't remind if they were modified or not.
+ * A mutable and updatable implementation of the {@link MutableAccount} interface, that tracks account updates since the
+ * creation of the updated it's linked to.
+ * <p>
+ * Contains {@code updateAccountTracker} for immediate set of balance in the world state. Note that in practice this
+ * only track the modified account values, but doesn't remind if they were modified or not.
  */
 public class UpdateTrackingAccount<A extends Account> implements MutableAccount, EvmAccount {
```

Comment:
Unfortunate that there's both this class `UpdateTrackingAccount` and a completely independent interface named `UpdateAccountTracker`.  Possibly that'll be confusing for readers in the future.